frontend/src/lib/utils.tsx

```diff
@@ -417,9 +417,8 @@ export function humanFriendlyLargeNumber(d: number): string {
     } else if (!isFinite(d)) {
         if (d > 0) {
             return 'inf'
-        } else {
-            return '-inf'
         }
+        return '-inf'
```

Comment:
is this based on master correctly 🤔 , iirc this if-else change should already be in master